examples/framesjs-starter/app/page.tsx

```diff
@@ -55,8 +55,8 @@ export default async function Home({
   // example: load the users credentials & check they have an NFT
 
   // Example with satori and sharp:
-  // const imageUrl = await 
-    (frameMessage);
+  // const imageUrl = await
+  frameMessage;
```

Comment:
whats this

Reply:
i think it was introduces somewhere else. removing